tests/integration/test_users.py

```diff
@@ -81,21 +81,52 @@ def test_refresh_user(client, test_database, admin_user):
     assert responseRefreshToken.status_code == 200
 
 
-def test_get_user_by_id(client, auth_headers, admin_user):
+def test_get_user_by_id(client, empty_test_db, create_admin_user, admin_header):
     """The get user by id route returns a successful response code."""
-    user = UserModel.find_by_email(admin_user.email)
-    response = client.get(f"/api/user/{user.id}", headers=auth_headers["admin"])
+    user = create_admin_user()
+    response = client.get(f"/api/user/{user.id}", headers=admin_header)
     assert response.status_code == 200
 
     """
     The server responds with an error if a non-existent user id
     is requested from the get user by id route.
     """
-    responseBadUserId = client.get("/api/user/000000", headers=auth_headers["admin"])
+    responseBadUserId = client.get("/api/user/000000", headers=admin_header)
     assert responseBadUserId.status_code == 404
     assert responseBadUserId.json == {"message": "User not found"}
 
 
+def test_get_pm_by_id(
+    client,
+    empty_test_db,
+    create_property_manager,
+    create_property,
+    create_lease,
+    admin_header,
+):
+    user = create_property_manager()
+    prop = create_property(manager_ids=[user.id])
+    lease_1 = create_lease(property=prop)
+    lease_2 = create_lease(property=prop)
+
+    response = client.get(f"/api/user/{user.id}", headers=admin_header)
+
+    property_list = response.json["properties"]
+    tenants_list = response.json["tenants"]
+
+    """The get user by id route returns a successful response code for PMs"""
+    assert response.status_code == 200
```

Comment:
I don't understand this comment. The admin_header made the request so shouldn't it be a successful response for admins? Am I missing something here?

This looks like an authorization test. Perhaps we can put this with the authorization tests. If a folder for authorization tests don't exist yet, perhaps now would be a good time to create one.

Reply:
@NickSchimek, sorry this was a confusing comment.  

I meant that it returned a successful response code when the user being queried was a PM. It's just testing that the endpoint now works when querying a PM user, since a previous PR broke that functionality (the endpoint was successful when querying admin, staff, etc, just not PMs).

Reply:
Thanks. Since this is testing the bug. I think this should be in a separate test function that is well named so that it doesn't get removed accidentally by someone in the future not knowing what it's here for.